### ISSUE TITLE
cant pair with the original “sixaxis”controller（no rumble）

### ISSUE THREAD

**Issue by maskmark** [TheOfficialFloW/MiniVitaTV#13]
the plugin works fine with my two dualshock3 controllers,but cannot even pair with my “sixaxis” controller come with early fat ps3 console. I can be sure this controller  is a original sony one cuz it can pair a psvtv wirelessly（which 3rd party controllers can never do）.but it now just act as 3rd party ones on my psv（3.65 enso）.

**Comment by seewood** [comment]
Same here, original Sony Sixaxis Controller CECHZC1E wont connect. Changed the MAC Adresse at the End from :BC to :BD but wont connect. My DS3 CECHZC2E wouldnt connect too, i dont know what im doin wrong. Every thing in my config on ur0 is correct. Checked 5 times. Both Controller work flawlessly on my PSTV on 3.60 ENSO

**Comment by maskmark** [comment]
> Same here, original Sony Sixaxis Controller CECHZC1E wont connect. Changed the MAC Adresse at the End from :BC to :BD but wont connect. My DS3 CECHZC2E wouldnt connect too, i dont know what im doin wrong. Every thing in my config on ur0 is correct. Checked 5 times. Both Controller work flawlessly on my PSTV on 3.60 ENSO
> 
> i tried with both Versions of the pair Tool. 0.3.1 and 0.1

Weird. All my three ds3 works with this plug-in . then probably your CECHZCE is a 3rd perty one , or you misunderstood something in the readme.txt ? like the controller's mac should be set to psv wifi mac+1,not original controller mac+1.

**Comment by seewood** [comment]
yeah i know that. my Vita has the Mac :BC at the end and with the sixaxis Tool i changed the Mac adress from my sixaxis Controller to the same as shown in settings under firmware but with :BD at the end. tested both Controller. DS3 and minivitatv.skprx are in ur0 and correctly edited in config.txt

**Comment by maskmark** [comment]
Finally! I got this solved. actually it's a work around. previously I had a sixaxis controller incompatible with my psv1000 but ds3 works . and none of them works with psv2000.
then I found the solution.
JUSTuse xerpi's ds3vita plug-in to pair the controller for the first time(all of them can be paired),and the delete xerpi's ds3vita ,use minivitatv and ds3 prx instead. NOW all     controllers pairs and works fine. though dunno why.

**Comment by maskmark** [comment]
yes I tried thrice on 4 controllers and 3 vita. but I think it's still a compatiblity bug, and I used xerpi's plugin to workaround. so I wont close this issue.maybe it could be fixed.
